gradle.properties

```diff
@@ -1,8 +1,6 @@
 org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m
 
-kotlin.mpp.enableGranularSourceSetsMetadata=true
 kotlin.mpp.enableCompatibilityMetadataVariant=true
-kotlin.native.enableDependencyPropagation=false
```

Comment:
The removed options are deprecated in Kotlin 1.6.20 per [release announcement](https://kotlinlang.org/docs/whatsnew1620.html#hierarchical-structure-support-for-multiplatform-projects).